torchtext/csrc/vectors.cpp

```diff
@@ -81,15 +281,13 @@ static auto vectors =
         .def("__len__", &Vectors::__len__)
         .def_pickle(
             // __setstate__
-            [](const c10::intrusive_ptr<Vectors> &self)
-                -> std::tuple<std::vector<std::string>, torch::Tensor,
-                              torch::Tensor> {
-              std::tuple<std::vector<std::string>, torch::Tensor, torch::Tensor>
-                  states(self->tokens_, self->vectors_, self->unk_tensor_);
-              return states;
+            [](const c10::intrusive_ptr<Vectors> &self) -> std::tuple<
+                Dict<std::string, int64_t>, torch::Tensor, torch::Tensor> {
```

Comment:
I'm not sure it's a good idea to serialize the dict vs. just the items, in terms of performance but also BC compatibility, since Dict is a JIT type.

Reply:
Basically forgot to merge in the latest changes from master. It should be fixed now